model/file.go

```diff
@@ -22,6 +22,36 @@ type FileUploadResponse struct {
 	ClientIds []string    `json:"client_ids"`
 }
 
+const (
+	FILE_SORT_BY_CREATED      = "CreateAt"
+	FILE_SORT_BY_SIZE         = "Size"
+	FILE_SORT_BY_USERNAME     = "Username"
+	FILE_SORT_BY_CHANNEL_NAME = "Name"
+
+	FILE_SORT_ORDER_ASCENDING  = "ASC"
+	FILE_SORT_ORDER_DESCENDING = "DESC"
+)
+
+// GetFilesOptions contains options for getting files
+type GetFilesOptions struct {
+	// UserIds optionally limits the files to those created by the given users.
+	UserIds []string
```

Comment:
Could we define `json` struct field tag in case we need to serialize this for any reason? (i.e. `json:"user_ids"`)

Reply:
I wasn't sure if this would require serialization. I'll add json tags.